.drone.yml

```diff
@@ -0,0 +1,38 @@
+---
+kind: pipeline
+name: default-image-push-pull
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: mirror-images
+  image: docker
+  volumes:
+  - name: docker
+    path: /var/run/docker.sock
+  environment:
+    PASSWORD:
+      from_secret: docker_password
+    USERNAME:
+      from_secret: docker_username
+  commands:
+  - docker login -u $USERNAME -p $PASSWORD
+  - cat images-list | ./docker-pull-pusher.sh
+  settings:
+    custom_dns: 1.1.1.1
+  when:
+    event:
+    - push
+
+volumes:
+- name: docker
+  host:
+    path: /var/run/docker.sock
+
+trigger:
```

Comment:
I believe we can remove this trigger exclude as that is a step in the other drone.yml.

Reply:
yes, removed it.